File: IHaveATheory/app/src/main/java/com/avoid/ihaveatheory/model/Item.java
package com.avoid.ihaveatheory.model;

public class Item {
    private ItemType itemType = null;
    private int qty = 0;

    public Item(ItemType itemType) {
        this.itemType = itemType;
    }

    public ItemType getItemType() {
        return itemType;
    }

    public int getQty() {
        return qty;
    }

    public void setQty(int qty) {
        this.qty = qty;
    }
}

File: app/src/main/java/com/avoid/ihaveatheory/model/item/Item.kt
package com.avoid.ihaveatheory.model.item

import java.io.Serializable

abstract class Item : Serializable {
    var itemType: ItemType
    var name: Int = 0
    var description: Int = 0
    var image: Int = 0

    constructor(itemType: ItemType, name: Int, description: Int, image: Int) {
        this.itemType = itemType
        this.name = name
        this.description = description
        this.image = image
    }
}
